docs/gen-docs/kyma_alpha_deploy.md

```diff
@@ -22,6 +22,7 @@ kyma alpha deploy [flags]
       --helm-timeout duration     Timeout for the underlying Helm client. (default 6m0s)
   -p, --profile string            Kyma deployment profile. If not specified, Kyma is installed with the default chart values. The supported profiles are: "evaluation", "production".
       --quit-timeout duration     Time after which the deployment is aborted. Worker goroutines may still be working in the background. This value must be greater than the value for cancel-timeout. (default 20m0s)
+      --skip-cert-import          Set to true if you want to skip importing kyma cert to the host machine.
```

Comment:
Pls confirm specifically how this value must be set - I assume similar to the --atomic flag? "Set `--atomic=false` to..."

--

```suggestion
      --skip-cert-import          Set `--skip-cert-import=true" to skip importing the Kyma certificate to the host machine.
```

(is it "the" or "a" certificate?

Reply:
To clarify, boolean flags can be set in 2 different ways:
- To set it to true, both `--atomic` and `--atomic=true` are equivalent.
- To set it t false, `--atomic=false` and not passing the flag at all are equivalent.